cmd/crio/wipe.go

```diff
@@ -18,6 +18,13 @@ var wipeCommand = &cli.Command{
 	Name:   "wipe",
 	Usage:  "wipe CRI-O's container and image storage",
 	Action: crioWipe,
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:    "force",
+			Aliases: []string{"f"},
+			Usage:   "force wipe by skipping the version check",
+		},
+	},
```

Comment:
The docs have to be re-generated for this via something like `sudo make clean all docs-generation completions`

Reply:
done, thanks!